app/controllers/exhibits_controller.rb

```diff
@@ -7,6 +7,11 @@ def ingest_members
     ExhibitProxy.new(@exhibit).reindex
   end
 
+  def destroy
+    @exhibit.resources.destroy_all
```

Comment:
Off-hand, I can't think of a reason to leave an orphaned database row. Cleaning up Solr, however, is probably best left to the downstream model, like you've done here.